modules/oe_translation_epoetry/src/EpoetryOngoingNewVersionRequestHandler.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_translation_epoetry;
+
+use Drupal\Component\Render\MarkupInterface;
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+
+/**
+ * Default handler for new version requests.
+ */
+class EpoetryOngoingNewVersionRequestHandler implements EpoetryOngoingNewVersionRequestHandlerInterface {
+
+  use StringTranslationTrait;
+
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * Constructs a EpoetryOngoingNewVersionRequestHandler.
+   *
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entityTypeManager
+   *   The entity type manager.
+   */
+  public function __construct(EntityTypeManagerInterface $entityTypeManager) {
+    $this->entityTypeManager = $entityTypeManager;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function canCreateRequest(TranslationRequestEpoetryInterface $request): bool {
+    // If the request doesn't have one of these statuses, we cannot make such
+    // request.
+    $access = in_array($request->getEpoetryRequestStatus(), [
+      TranslationRequestEpoetryInterface::STATUS_REQUEST_ACCEPTED,
+    ]);
+
```

Comment:
@22Alexandra If the request is rejected, we cannot make a new request for the same version. You can see the docs. Which makes sense for update requests for ongoing, but what if they reject, we change the content and then send a new version request...I will talk to DGT. For now this can stay like this because this access handler is used on the Update requests (for ongoing)

Reply:
@22Alexandra @hernani Discussed with them. If the thing gets rejected, we will use a different request type, for the same version. Will be another ticket. So we can leave this as is.